administrator/components/com_associations/models/associations.php

```diff
@@ -224,6 +232,11 @@ protected function getListQuery()
 		}
 
 		// Add the list ordering clause.
+		if ($table != '#__menu')
+		{
+			$this->setState('list.ordering', 'ordering');
+		}
```

Comment:
The problem is the following:
If we display menu items and sort by menu ordering and then switch to another item than menu items (articles, categories), we get an sql error. This is the only way I found to avoid this issue. Better way welcome.